fares-data-build-tool/fdbt-site
# Feature/tfn 484 forgotten password reset

# Description

The ability for the user to reset their password via the email link. https://infinityworks.atlassian.net/browse/TFN-483

Related pr for aws cognito https://github.com/fares-data-build-tool/fdbt-aws/pull/10

# Testing instructions

- Make sure you are created in the user pool under tfn-test within aws
- Once you have a confirmed account you can go into the userpool and select your user and press 'Reset password'
- Go to your email client and click on the link. As this on test copy everything after https://tfn-test.infinityworks.com/ and paste into your browser replacing the test url with localhost:5555
- Now you can test the scenairos as highlighted in the ticket within Jira (link above)
- Also another test is that if you have changed your password then the user cannot reuse the same link.

<img width="685" alt="Screenshot 2020-06-09 at 14 22 24" src="https://user-images.githubusercontent.com/1183416/84153141-6ae08180-aa5d-11ea-90ca-84c9549cac36.png">

# Type of change


# Checklist:
